pkg/util/hack.go

```diff
@@ -28,3 +30,11 @@ func Slice(s string) (b []byte) {
 	pbytes.Cap = pstring.Len
 	return
 }
+
+// VarcharOnly converts char to varchar if ENV VAR VARCHAR_ONLY=true
+func VarcharOnly(query string) string {
+	if os.Getenv("VARCHAR_ONLY") == "true" {
+		return strings.Replace(strings.Replace(query, " CHAR", " VARCHAR", -1), " char", " varchar", -1)
```

Comment:
Emmm this looks stupid. Why not just replacing them in code?

Reply:
In case we need to support the char type in the future we don't need to modify any code.

Reply:
In that case I would rather modify the code 😄

Reply:
I prefer not to introduce any environment variable here. Too tricky.